libs/blocks/quiz/quiz.js

```diff
@@ -87,6 +94,11 @@ const App = () => {
       handleResultFlow(transformToFlowData(userSelection));
     } else {
       setUserFlow(nextQuizViews);
+      if (nextQuizViews[0] === 'q-rather' || nextQuizViews[0] === 'q-customer') {
```

Comment:
@fullcolorcoder - `q-rather` and `q-customer` should not be hard coded. These are dynamic values that are authorable.